### PR TITLE
tweaks

### LINKED ISSUE #903: Batch upload success alert treats children with enrollment changes as separate enrollments
When I upload a record with an infant/toddler enrollment and a preschool enrollment, the alert says I've uploaded 1 active enrollment and 1 withdrawn enrollment.

Desired behavior is for that record to count as 1 active enrollment.

### LINKED ISSUE #909: Determine home page display for multi-org user
Currently, we have an h1 that says "Hello, [User name]!" and an h2 that says "Sites you manage at [org name]". We need to figure out how to display the h2 for users that are associated with multiple organizations. Displaying the list of sites for each organization below it could also be long and messy.

### LINKED ISSUE #930: The Back button on the withdrawn roster or past month's roster should go to the current roster instead of the home page.

### PR DESCRIPTION
**Pull request by jlhogan**
closes #930 
closes #903 
closes #909 
fixes broken link to data-requirements from homepage
does a tiny bit of consistency-adding to headers/spacing/back buttons 
renames GettingStarted to Home and Home to Landing

### CODE REVIEW

File: client/src/components/DataDefinitionsTable/DataDefinitionsTable.tsx [hunk 1]
```diff
@@ -68,11 +68,12 @@ const DataDefinitionsTable: React.FC<DataDefinitionsTableProps> = ({
     );
   }
 
-  if (user) {
+  if (user && filteredColumnMetadata.length) {
```
Comment (jlhogan) [inline client/src/components/DataDefinitionsTable/DataDefinitionsTable.tsx:71]: this is the fix for the data requirements link

File: client/src/containers/Roster/Roster.tsx [hunk 2]
```diff
@@ -159,8 +159,11 @@ const Roster: React.FC = () => {
     <>
       <div className="Roster grid-container">
         <BackButton
-          text="Back to getting started"
-          location="/getting-started"
+          location={
+            query.month || query.withdrawn
```
Comment (jlhogan) [inline client/src/containers/Roster/Roster.tsx:163]: this implements 930

File: src/routes/enrollmentReports.ts [hunk 3]
```diff
@@ -134,16 +134,14 @@ enrollmentReportsRouter.post(
         let numActive = 0,
           numWithdrawn = 0;
         reportChildren.forEach((child) => {
-          child.enrollments?.forEach((e) => {
-            if (e.exit) numWithdrawn += 1;
-            else numActive += 1;
-          });
+          if (child.enrollments?.every((e) => e.exit)) numWithdrawn += 1;
```
Comment (jlhogan) [inline src/routes/enrollmentReports.ts:137]: this fixes #903

File: client/src/containers/Home/Home.tsx [hunk 4]
```diff
@@ -1,147 +1,140 @@
...
+      <div className="grid-row grid-gap">
+        <div className="tablet:grid-col-8">
+          <h1 ref={h1Ref}>Hello {user?.firstName}!</h1>
+          <p>
+            ECE Reporter will allow you to share your state-funded enrollment
+            data with the Office of Early Childhood (OEC). OEC uses this data to
+            make informed program and policy decisions affecting young children
+            and families.
+          </p>
+          {(user?.organizations || []).map((org) => (
```
Comment (jlhogan) [inline client/src/containers/Home/Home.tsx:24]: this deals with #909

**Review by bamader — APPROVED** [review]

File: client/src/containers/Home/Home.test.tsx [hunk 5]
```diff
@@ -2,14 +2,37 @@ import React from 'react';
...
+import { User } from '../../shared/models';
+import moment from 'moment';
+
+const testUser = {
+  firstName: 'Voldy',
+  organizations: [
+    { id: 1, providerName: 'Mort School For Dark-Inclined Wizards' },
+  ],
+  sites: [
+    { id: 1, siteName: 'Students of Salazaar', organizationId: 1 },
```
Comment (mmazanec22) [inline client/src/containers/Home/Home.test.tsx:14]: Lol yes!

**Review by mmazanec22 — APPROVED** [review]

**Pull request merged**